woocommerce/woocommerce-gutenberg-products-block
# Add inner wrapper div in 'td.wc-block-cart-item__product'

Adding this wrapper element is necessary in order for us to be able to "fake" item grouping in a theme-agnostic manner: 

https://user-images.githubusercontent.com/1783726/118831630-97855600-b8c8-11eb-8906-c7f9ada769e7.png

Discussed in #4241 

This is a CSS requirement -- it is not possible to reliably obtain the result seen above (let alone adjust it via breakpoints) without a wrapper element.

#### Other Checks


### Testing

#### Automated Tests

### Manual Testing

How to test the changes in this Pull Request:

1. Add two different products to the cart.
2. Inspect the cart table when the small/medium/large CSS breakpoints are active.
3. Confirm that no visual issues are caused by the additional `div`.

### User Facing Testing

These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).

1.
2.
3.



### Performance Impact

None

### Changelog

> Wrapped cart item product contents in inner div.
